### PR TITLE
Support region identifiers

### PR DESCRIPTION
**Pull request by Mehrnaz-Charkhchi**
This PR contains 
 - [ENSWBSITES-2256](https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-2256) You can find the updates in the related [commit](https://github.com/Ensembl/ensembl-web-metadata-api/commit/6d022837838fae6e9fe023822755b1350699828a).
 - Python black formatted code

### CODE REVIEW

File: app/api/models/region_validation.py [hunk 1]
```diff
@@ -26,65 +25,74 @@ class RegionValidation(BaseModel):
...
-                start, end = rn_coords[1].split("-")
-                start = start.replace(",","")
-                end = end.replace(",","")
+            region_coordinates = region_input.split(":")
+            region = region_coordinates[0]
+            start, end = region_coordinates[1].split("-")
+            start = start.replace(",", "")
+            end = end.replace(",", "")
+        except IndexError:
+            pass
```
Comment (kamaldodiya) [inline app/api/models/region_validation.py:52]: logger.debug() ?

**Review by kamaldodiya — COMMENTED** [review]
Can you please add few example for validate_location and region_details post this changes ?

File: app/api/models/region_validation.py (hunk 1 above)
Comment (Mehrnaz-Charkhchi) [reply app/api/models/region_validation.py:52]: It's an expected error that I want to purposely pass . we don't need a logger for that, I kept the one in the next line for unexpected errors.

**Comment by Mehrnaz-Charkhchi** [comment]
> Can you please add few example for validate_location and region_details post this changes ?

@kamaldodiya Here is an example url: http://0.0.0.0:8014/api/metadata/validate_location?genome_id=a7335667-93e7-11ec-a39d-005056b38ce3&location=21
Screenshot 2024-01-31 at 14 20 20

**Pull request merged**